restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -236,7 +237,7 @@
 	"formProvideConfigurationLabel": "Proporcione las opciones de configuración para conectar esta aplicación a el entorno del usuario final.",

```

Comment:
Grammar error found. Change for:
"formProvideConfigurationLabel": "Proporcione las opciones de configuración para conectar esta aplicación al entorno del usuario final.",